components/shopify_partner/package.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Thanks for adding this. When you merge to `master`, we should publish the package automatically.

Can you DM me your `npm` username? I'll add you to our `pipedream` npm organization.